phpunit/phpunit/4.7/tests/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use App\Kernel;
+
+/*
+ * Environment variables can also be specified in phpunit.xml.dist.
```

Comment:
This was a copy paste, but needs double check / test as i'm not sure if this is completely true.